packages/fxa-settings/src/components/en.ftl

```diff
@@ -21,9 +21,13 @@
 # This is used to refer to a user's account, e.g. "update your Firefox account ..."
 -product-firefox-account = Firefox account
 product-mozilla-vpn = Mozilla VPN
+product-mozilla-mdn = Mozilla MDN
```

Comment:
@bcolsson I was going to ask if these should be [terms instead](https://projectfluent.org/fluent/guide/terms.html), but it looks like terms shouldn't be used stand-alone and should only be references in other messages, so this seems right. Is it safe to reference these _like_ terms later, e.g. `You have a { product-mozilla-mdn } subscription`?

I'm not sure if we've determined we shouldn't translate "Mozilla MDN" or "Mozilla Hubs" etc. - if that's the case we don't need these wrapped in a `Localized` component, but looks like we previously had `Mozilla VPN` and `Mozilla Monitor` set up this way, so 🤷‍♀️

Reply:
What Bryan wrote is absolutely accurate. We want to use brand for our product names (not feature names), `Mozilla VPN` being a good example. The [brandings.ftl file](https://searchfox.org/mozilla-central/source/browser/locales/en-US/browser/branding/brandings.ftl) in Firefox is a good example.

In this specific case, I would argue that "Mozilla MDN" is not a thing? MDN = Mozilla Developer Network, there is no "Mozilla MDN", there is "Mozilla's MDN" at best. On mozilla.org it's called out as "MDN Plus" in the Products menu.

Reply:
Thank you all for the feedback. I've tagged Vesta on the Jira ticket to help take a look at the points you've raised regarding naming and translation, and I've tagged you on the ticket as well.

Regarding the other changes, I've made the entries terms, and added a separate translation string. Please let me know if I missed something.